### ISSUE TITLE
Interleave data breaks when dealing with a combination of (Nt,1) and (Nt,2) conductance models

### ISSUE THREAD

**Issue by ma-sadeghi** [PMEAL/OpenPNM#1676]
**Describe the bug**
Suppose you have two geometries, one involves a conductance model that returns (Nt,1), and the other (Nt,2). Calling `throat.conductance` on the phase object fails.

**To Reproduce**
Steps to reproduce the behavior, ideally a code snippet that only reproduces the bug:
```python
import numpy as np
import openpnm as op


np.random.seed(10)
ws = op.Workspace()

net = op.network.Cubic(shape=[5, 1, 1])

geom2 = op.geometry.SpheresAndCylinders(network=net, pores=net.Ps, throats=net.Ts[:-1])
geom1 = op.geometry.SpheresAndCylinders(network=net, throats=net.Ts[-1])

air = op.phases.Air(network=net)

phys2 = op.physics.Standard(network=net, phase=air, geometry=geom1)
phys1 = op.physics.Standard(network=net, phase=air, geometry=geom2)

mod = op.models.physics.ad_dif_conductance.ad_dif
phys2.add_model("throat.foo", model=mod)
phys1["throat.foo"] = 1.234

g = air["throat.foo"]
print(g)
```
```python
Traceback (most recent call last):

  File "/home/amin/Code/scripts/temp.py", line 22, in <module>
    g = air["throat.foo"]

  File "/home/amin/Code/OpenPNM/openpnm/phases/GenericPhase.py", line 92, in __getitem__
    vals = super().__getitem__(key)

  File "/home/amin/Code/OpenPNM/openpnm/core/Base.py", line 27, in __getitem__
    vals = super().__getitem__(key)

  File "/home/amin/Code/OpenPNM/openpnm/core/Base.py", line 254, in __getitem__
    vals = self.interleave_data(key)

  File "/home/amin/Code/OpenPNM/openpnm/core/Base.py", line 810, in interleave_data
    temp_arr[inds] = vals

ValueError: shape mismatch: value array of shape (3,)  could not be broadcast to indexing result of shape (3,2)
```

**Comment by jgostick** [comment]
We should probably enforce all conductance models to be Nt-by-2.  Also, should think about capillary pressure related things, since percolation could be direction dependent...as in graphs can be directed or undirected.

**Comment by jgostick** [comment]
This was re-raised in issue #2135, which is more specific, so closing this

**Comment by ma-sadeghi** [comment]
@jgostick It seems that this is still not fixed (or maybe we didn't want to fix it? and just enforce all conductance arrays to be Nt by 2?)

**Comment by jgostick** [comment]
But I even wrote a test?

**Comment by jgostick** [comment]
So, I fixed the case of [Nt, 1] + [Nt, 2], as well as [Nt, 2] + [Nt, 1], BUT I didn't fix the case for [Nt, ] + [Nt, 2] or [Nt, 2] + [Nt, ].

**Comment by jgostick** [comment]
This is now properly fixed in #2230
